src/main/java/io/jenkins/plugins/opentelemetry/authentication/BearerTokenAuthentication.java

```diff
@@ -70,6 +71,12 @@ public void configure(@Nonnull OtlpGrpcSpanExporterBuilder spanExporterBuilder)
         spanExporterBuilder.addHeader("Authorization", "Bearer " + this.getAuthenticationHeaderValue());
     }
 
+    @Override
+    public void enrichOtelEnvironmentVariables(Map<String, String> environmentVariables) {
+        // TODO don't overwrite OTEL_EXPORTER_OTLP_HEADERS if already defined, just append to it
+        environmentVariables.put("OTEL_EXPORTER_OTLP_HEADERS", "Authorization=Bearer " + this.getAuthenticationHeaderValue());
```

Comment:
See https://github.com/open-telemetry/opentelemetry-python/issues/1851

```suggestion
        environmentVariables.put("OTEL_EXPORTER_OTLP_HEADERS", "authorization=Bearer " + this.getAuthenticationHeaderValue());
```